frontend/src/components/pages/ViewDonations.tsx

```diff
@@ -70,6 +71,7 @@ const ViewDonations = ({
               })}
             </Text>
             <DatePicker
+              minDate={new Date().setDate(today.getDate())}

```

Comment:
nit: could we set the `minDate` to the `startTime` date to prevent users from picking an end date that comes after the start date?

Reply:
thanks for catching that!